### PR TITLE
Add scheduler

### PR DESCRIPTION
**Pull request by hackinghackers**

### CODE REVIEW

**Comment by hackinghackers** [comment]
still have questions on the formatting and how runningAI is called

File: backend/main.py [hunk 1]
```diff
@@ -1,25 +1,144 @@
...
 
-    api_key = os.environ.get('TEST')
-    print(api_key)
+    logging.debug(f"Sending data for {meal}")
+
+    with open("counter.txt", "r") as f:
+        counter = int(f.read())
+    logging.debug(f"counter={counter}")
+
+    data = {
```
Comment (retz8) [inline backend/main.py:121]: Please add a comment here like, "formatting data to final JSON format".

File: backend/main.py [hunk 2]
```diff
@@ -1,25 +1,144 @@
...
+
+    if "--run" not in sys.argv:
+        with open("counter.txt", "w") as f:
+            f.write("0")
+
+        with open("data.json", "w") as f:
+            json.dump({}, f)
+
+        with CronTab(user="root") as cron:
+            breakfast = cron.new(
```
Comment (retz8) [inline backend/main.py:43]: 1) Just to clarify, CronTab is a Python library that allows a computer to run the command automatically. Then, do we need to keep the computer turned on? 
2) Please add a comment here like, "CronTrab Scheduler Configuration".

File: backend/main.py [hunk 3]
```diff
@@ -1,25 +1,144 @@
-# command: python main.py
+#!/usr/bin/env python3
```
Comment (retz8) [inline backend/main.py:1]: What will be the final command when we run the code on Jetson?

**Review by retz8 — COMMENTED** [review]
Hi Andy, thank you for the pull request. I reviewed the code. Please check my comments. Most of them are about adding comments to make your code more readable. I also have several questions, so please discuss them with Mr. Tang during club time.

File: backend/main.py [hunk 4]
```diff
@@ -1,25 +1,144 @@
...
+
     # ------------------
     # Data measurement logic + call runningAI.py
     # ------------------
 
+    logging.info(f"Running AI for {meal}")
+    start_time = datetime.datetime.now()
+
+    results = runningAI.run_till(meal_end_time[meal])
+
```
Comment (retz8) [inline backend/main.py:105]: What is your expected `results` format here? Is it an array of numbers like [5, 1, 3, 2, 1, 5, 3, 2, 3, 1, 3]?

**Review by retz8 — COMMENTED** [review]
.

**Review by retz8 — APPROVED** [review]
Thank you Andy. We will test the whole code on Wednesday at club time.

**Pull request merged**